src/OpenLoco/Map/TileManager.cpp

```diff
@@ -91,12 +92,45 @@ namespace OpenLoco::Map::TileManager
         TileManager::updateTilePointers();
     }
 
+    // Note: Must be past the last tile flag
+    static void markElementAsFree(TileElement& element)
+    {
+        element.setBaseZ(255);
+        if (element.next() == *_elementsEnd)
```

Comment:
can you use `if (element.isLast())` here?

Reply:
No, the end pointer is the very last element in the whole allocation of map elements. Is last is the last element for the tile coordinate.